lib/coap.js

```diff
@@ -32,16 +32,16 @@ function CoAP(opts, done) {
   this._persistence = opts.ponte.persistence;
   this._broker = opts.ponte.broker;
   this._ponte = opts.ponte;
-  
-  if (typeof opts.authenticate === "function") {
+
+  if (helper.isFunction(opts.authenticate)) {
```

Comment:
can you please do a dummy commit, or do an `--amend` changing something silly? I do not think the hook gets called when closing/opening.

What Ponte is missing from a QEST perspective is an UI. Would you like to contribute?
I am working making better all the pubsub beneath it.